lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -60,6 +61,24 @@ function dosomething_northstar_verify_user($credentials) {
     'data' => json_encode($credentials),
   ]);
 
+  // Add to request log
+  if(variable_get('dosomething_northstar_log')) {
+    // Don't log plaintext password.
+    $credentials['password'] = '*****';
+
+    db_insert('dosomething_northstar_request_log')
+      ->fields([
+        'op' => 'verify user',
+        'uid' => $user->uid,
+        'user_lang' => $user->language,
+        'user_country' => $user->field_address[LANGUAGE_NONE][0]['country'],
```

Comment:
hmm, I wonder if it's also helpful to see if this differs from the fastly country code to help with #6159